src/controllers/matomo.js

```diff
@@ -47,10 +47,8 @@ export default async (req, res) => {
 
   if (!req.query?.format) {
     params = `${params}&format=JSON`;
-  } else if (typeof req.query.format !== 'string') {
+  } else if (typeof req.query.format !== 'string' || (typeof req.query.format === 'string' && req.query.format.toLowerCase() !== 'json')) {
```

Comment:
Se la valutazione del connettivo `||` arriva al secondo membro allora è garantito che il primo membro è falso, quindi il secondo check è inutile.
```suggestion
  } else if (typeof req.query.format !== 'string' || req.query.format.toLowerCase() !== 'json') {
```

Reply:
Hai ragione, ho modificato il controllo